dependabot.yml

```
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  # For the `main` branch
+  - package-ecosystem: "yarn"
+    directory: "/"     
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)